src/translation-store.ts

```diff
@@ -6,24 +6,45 @@ export interface TranslationService {
     i18n: Readable<i18n>;
 }
 
+export const isLoading = writable(false);
```

Comment:
probably needs to be set to true by default?

Reply:
that's right, updated the implementation :)
Thank you